html/partials/50x.shtml

```diff
@@ -1,2 +1,3 @@
-<p>Sorry, the page you are looking for is currently unavailable.</p>
-<p>Please try again later.</p>
+<h2>An error occurred</h2>
+<p>The page you’re looking for isn’t available.</p>
```

Comment:
We should probably change this thinking about it. This page is really a catch-all.
It's difficult to know what to write.
Perhaps "We were unable to fulfil your request."?
@easternbloc: Any ideas?